peer/core.yaml

```diff
@@ -413,7 +413,7 @@ security:
     multithreading:
       enabled: false
 
-    # Confidentiality protocol version could be 1.1 or 1.2
+    # Confidentiality protocol version could supported: 1.2
```

Comment:
"Confidentiality protocol version could supported: 1.2" should be "Confidentiality protocol versions supported: 1.2"